lib/ui/Editor.js

```diff
@@ -621,6 +626,14 @@ Editor.prototype._updateCursor = function () {
     self.screen.program.hideCursor();
   }
 };
+Editor.prototype._clipCursorToContent = function (cursor, content) {
```

Comment:
Remove this `content` param and use `self.textBuf.getEndPosition()` which will return a `Point` instead.